pkgs/applications/window-managers/newm/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib,
+fetchFromGitHub,
```

Comment:
Each line should start with comma, not end with it.

```suggestion
{ lib
, fetchFromGitHub
```

And do the same below